edge/pkg/edged/edged.go

```diff
@@ -1088,14 +1058,13 @@ func (e *edged) consumePodAddition(namespacedName *types.NamespacedName) error {
 	// Fetch the pull secrets for the pod
 	secrets := e.getImagePullSecretsForPod(pod)
 
-	podUID := pod.GetUID()
 	t, ok := e.podLastSyncTime.Load(podUID)
 	if !ok {
 		t = time.Time{}
 	}
 	curPodStatus, err := e.podCache.GetNewerThan(podUID, t.(time.Time))
 	if err != nil {
-		return fmt.Errorf("pod %s cache newer failed: %v", podName, err)
+		return fmt.Errorf("pod %s cache newer failed: %v", pod.Name, err)
```

Comment:
we print Name and UID meanwhile, is this need unifying?

Reply:
use format.Pod(pod) that contains pod name namespace and uid